kalite/distributed/management/commands/setup.py

```diff
@@ -250,11 +269,25 @@ def handle(self, *args, **options):
         call_command("syncdb", interactive=False, verbosity=options.get("verbosity"))
         call_command("migrate", merge=True, verbosity=options.get("verbosity"))
 
+        # download assessment items
         # This can take a long time and lead to Travis stalling. None of this is required for tests.
-        if not settings.RUNNING_IN_TRAVIS:
-            # download assessment items
-            # TODO-BLOCKER (aron): do not hardcode this, and put in the proper URL once we have a redirect set up
-            call_command("unpack_assessment_zip", "http://eslgenie.com/media/assessment_item_resources.zip")
+        if not settings.RUNNING_IN_TRAVIS and options['interactive']:
```

Comment:
I didn't want to assume that the user was networked in non-interactive mode, but you're right that adding a flag to force a download would be a good idea.